app/controllers/vacancies_controller.rb

```diff
@@ -48,6 +59,24 @@ def set_headers
     response.set_header("X-Robots-Tag", "noarchive")
   end
 
+  def campaign_search_params
+    params.permit(:email_name, :email_postcode, :email_location, :email_radius, :email_jobrole, :email_subject,
+                  :email_phase, :email_ECT, :email_fulltime, :email_parttime, :email_jobshare, :email_contact)
+          .tap do |campaign_params|
+      campaign_params[:location] = campaign_params.delete(:email_location)
+      campaign_params[:radius] = campaign_params.delete(:email_radius)
```

Comment:
I think we are supposed to pre-select 15 mile radius if it isn't specified (which it probably won't be?)